### PR TITLE
Remove plugin manifest generator

### PR DESCRIPTION
**Pull request by sashayakovtseva**
## Description of the Pull Request (PR):

Instead of a separate go program, generate manifest in singularity.

#### Before submitting a PR, make sure you have done the following:

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: internal/app/singularity/plugin_compile_linux.go [hunk 1]
```diff
@@ -123,42 +125,35 @@ func buildPlugin(sourceDir, buildTags string) (string, error) {
...
-	if err != nil {
-		return "", errors.New("singularity source directory not found")
-	}
-
+func generateManifest(sourceDir string) (string, error) {
 	in := pluginObjPath(sourceDir)
 	out := pluginManifestPath(sourceDir)
 
-	goTool, err := exec.LookPath("go")
+	sylog.Debugf("Opening %q", in)
```
Comment (pisarukv) [inline internal/app/singularity/plugin_compile_linux.go]: do we need that debug line? it doesn't seem very informative.

File: internal/app/singularity/plugin_compile_linux.go [hunk 2]
```diff
@@ -62,11 +64,11 @@ func CompilePlugin(sourceDir, destSif, buildTags string) error {
 	// build plugin object using go build
 	_, err := buildPlugin(sourceDir, buildTags)
 	if err != nil {
-		return fmt.Errorf("while building plugin .so: %s", err)
+		return fmt.Errorf("while building plugin .so: %v", err)
```
Comment (WestleyK) [inline internal/app/singularity/plugin_compile_linux.go:67]: Why change this to `%v`?

**Review by WestleyK — COMMENTED** [review]
>**WestleyK** commented on this PR as **WestleyK**

Looks fine to me, just one question...

File: internal/app/singularity/plugin_compile_linux.go (hunk 2 above)
Comment (sashayakovtseva) [reply internal/app/singularity/plugin_compile_linux.go:67]: Here it will not change the output, a matter of habit rather.

**Review by dtrudg — APPROVED** [review]

**Pull request merged**